### PR TITLE
RmNode utility and minor UI fixes

### PR DESCRIPTION
**Pull request by tal-az**

### CODE REVIEW

File: controllers/agents.js [hunk 1]
```diff
@@ -1,6 +1,14 @@
+const { response } = require('express');
 const fleetService = require('../services/fleet');
 const systemService = require('../services/system');
 
+fleetPlatformString = {"debian": "ubuntu", "kali": "ubuntu", "ubuntu" : "ubuntu", "centos" : "centos", "rhel" : "centos"}
+os_to_delete_script = {"ubuntu": "removeFleetUbuntu.sh", "centos" : "RemoveFleetCentos.sh"}
```
Comment (maziland) [inline controllers/agents.js]: Keep on camelCaseForYourVariables :)

File: controllers/agents.js [hunk 2]
```diff
@@ -35,7 +43,51 @@ async function addNode(req, res) {
...
+    }
+}
+
+async function rmNode(req, res) {
+    try {
+        const hostId = req.params.hostId;
+        if (!hostId) {
+            return res.status(400).send({ error: 'hostID parameter is required' });
+        }
+        console.log("[*] Host to be removed: " + hostId)
```
Comment (maziland) [inline controllers/agents.js]: No need to log to the users console

File: controllers/endpoints.js [hunk 3]
```diff
@@ -92,6 +92,32 @@ async function addNode(req, res) {
     }
 }
 
+async function rmNode(req, res) {
+    try {
+        console.log('Received request to remove host with data:', req.body); // Add log here
```
Comment (maziland) [inline controllers/endpoints.js:96]: Can be deleted also

File: public/js/endpoints.js [hunk 4]
```diff
@@ -16,6 +15,9 @@ $(document).ready(function() {
             case 'uninstall_chipsec':
                 api_url = `/api/chipsec/uninstall/${endpoint_host_id}`;
                 break;
+            case 'delete_host':
+                api_url = `/api/agents/rmNode/${endpoint_host_id}`;
```
Comment (maziland) [inline public/js/endpoints.js:19]: I think it's better to follow urls without capital letters. (it's not common) `/api/agents/delete` is more RESTful
I noticed that we have got a few routes with capitals, so consider changing it some time in the future.

File: routes/router.js [hunk 5]
```diff
@@ -27,6 +27,11 @@ router.post("/api/agents/addNode/:osType/:hostId", (req, res, next) => {
     next();
 }, agentsController.addNode);
 
+router.get("/api/agents/rmNode/:hostId", (req, res, next) => {
+    console.log('rmNode route called');
+    next();
```
Comment (maziland) [inline routes/router.js]: Remove if it works for now

File: scripts/RemoveFleetCentos.sh
Comment (maziland) [inline scripts/RemoveFleetCentos.sh:1]: I'm feeling katnuni, but files should be snake_case 😆 
Dont hate me

File: scripts/whoami.sh
Comment (maziland) [inline scripts/whoami.sh:1]: Still relavant?

File: public/js/endpoints.js [hunk 6]
```diff
@@ -137,12 +139,52 @@ addNodeForm.addEventListener('submit', function(event) {
...
+
+        newToast.classList.add('toast-error');
+        newToast.querySelector('.toast-header strong').textContent = 'Action Failed';
+        newToast.querySelector('.toast-body').textContent = 'An error occurred. Please try again.';
+
+        container.append(newToast);
+        const toast = bootstrap.Toast.getOrCreateInstance(newToast);
+        toast.show();
+
+        console.log(error);
```
Comment (maziland) [inline public/js/endpoints.js:172]: fix identation

Comment (maziland) [inline public/js/endpoints.js:172]: error is not defined here

**Review by maziland — CHANGES_REQUESTED** [review]
I wrote some potential improvements, Looks good and functional :)

File: public/js/endpoints.js (hunk 4 above)
Comment (tal-az) [reply public/js/endpoints.js:19]: next time :)

File: scripts/RemoveFleetCentos.sh
Comment (tal-az) [reply scripts/RemoveFleetCentos.sh:1]: done deal

File: scripts/whoami.sh
Comment (tal-az) [reply scripts/whoami.sh:1]: removed

**Review by maziland — APPROVED** [review]

**Pull request merged**